src/Security/Authorization/Core/src/DefaultAuthorizationService.cs

```diff
@@ -75,6 +132,9 @@ public virtual async Task<AuthorizationResult> AuthorizeAsync(ClaimsPrincipal us
         }
 
         var result = _evaluator.Evaluate(authContext);
+
+        _metrics?.AuthorizedRequest(policyName, result);
```

Comment:
This should handle error situations, e.g. policy name can't be found, error from handler

Should something be recorded about the user? A flag for anonymous vs authenticated user.

Reply:
> I think we should also track when the `AuthorizationMiddleware` skips calling `AuthorizeAsync` due to `IAllowAnonymous` metadata.

We discussed this offline and decided that this would be better addressed in a follow-up.